perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/MembersManagerBlImpl.java

```diff
@@ -938,6 +970,48 @@ public RichMember getRichMemberWithAttributes(PerunSession sess, Member member)
 		return richMembers;
 	}
 
+	//also adds group-member attributes
+	public List<RichMember> convertMembersToRichMembersWithAttributes(PerunSession sess, Group group, List<RichMember> richMembers, List<AttributeDefinition> attrsDef)  throws InternalErrorException, WrongAttributeAssignmentException {
+		List<AttributeDefinition> usersAttributesDef = new ArrayList<AttributeDefinition>();
+		List<AttributeDefinition> membersAttributesDef = new ArrayList<AttributeDefinition>();
+		List<AttributeDefinition> groupAttributesDef = new ArrayList<AttributeDefinition>();
```

Comment:
Better use "member_group" naming consistently, since we have also "group" namespace and it's confusing, if it contains group attributes or not. So, something lik: ``memberGroupAttributesDef``